NewForce-Cohort-5/tabloidfullstack-spring-of-deception
# View All User Profiles

# Description

I added the ability to view all user profiles including the UserName, the users full name, and the user type. I also added a link in the nav bar to navigate to the user profile page.

## Type of change

[Check Trello Ticket #23](https://trello.com/b/IVGnzwap/spring-of-deception)

- [] New feature (non-breaking change which adds functionality)

# Testing Instructions

1. git add and git commit all changes in your current branch.
2. git checkout main
3. git fetch -a
4. git checkout np-view-all-user-profiles
5. Navigate to the root directory for this repo (directory-to)/tabloidfullstack-spring-of-deception
6. Start Tabloid.sln
7. Make sure your db is created with seed data (Skip to 9 if this is already done)
8. Run [01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/01_Tabloid_Create_DB.sql) and [02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/02_Tabloid_Seed_Data.sql) queries in Visual Studio
9. Start the server in Debug Mode F5
10. Log in as admin foo@bar.com (or any user)
11. Navigate to the User Profile link
12. Navigate to post details
13. User should see all user profiles




# Checklist:

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] My changes generate no new warnings or errors
- [] I have added test instructions that prove my fix is effective or that my feature works
